### PR TITLE
Fix Multiline to avoid warning from React

### PR DESCRIPTION
**Pull request by njt1982**
Addresses #882 

Works On My Machine™️

### CODE REVIEW

File: packages/themed/src/components/Textarea/styled-components/Input.tsx [hunk 1]
```diff
@@ -5,8 +5,6 @@ export default styled(
   TextInput,
   {
     p: '$2',
-    //@ts-ignore
-    multiline: true,
```
Comment (Viraj-10) [inline packages/themed/src/components/Textarea/styled-components/Input.tsx:9]: Hey @njt1982, `multiline` is a part of a prop and not the styling that's why we were getting this issue.
moving it inside props that should fix it.
```
props: {
      placeholderTextColor: '$textLight500',
      multiline: true,
    },
```

**Review by Viraj-10 — CHANGES_REQUESTED** [review]

File: packages/themed/src/components/Textarea/styled-components/Input.tsx (hunk 1 above)
Comment (njt1982) [reply packages/themed/src/components/Textarea/styled-components/Input.tsx:9]: I’ll make that change. Thanks!

Comment (njt1982) [reply packages/themed/src/components/Textarea/styled-components/Input.tsx:9]: @Viraj-10 done - I've left the `//@ts-ignore` comment, not sure if its still needed.

I've manually applied this change to my local node_modules checkout for my project and it seems happy enough...

Comment (Viraj-10) [reply packages/themed/src/components/Textarea/styled-components/Input.tsx:9]: Awesome 🚀 .

**Review by Viraj-10 — APPROVED** [review]

**Pull request merged**